runtimes/python3.nix

```diff
@@ -1,27 +1,24 @@
-{pkgs, piston, ...}:
-let
-    pkg = pkgs.python3;
-in piston.mkRuntime {
+{ pkgs, piston, ... }:
+let basePkg = pkgs.python3;
+in piston.mkRuntime (libraries:
```

Comment:
Could we possibly make this `piston.mkRuntimePackages` or similar, just so if packages aren't required we can use the old format?